Document the `kube_namespace` option of `ac_include`/`ac_exclude` (#7067)
DataDog/documentation: content/en/agent/autodiscovery/management.md

## Exclude containers

> Exclude containers from the Agent Autodiscovery perimeter with an exclude rule based on their `name`, `image` or `kube_namespace` to collect **NO DATA** out of it. If a container matches an exclude rule, it won't be included unless it first matches an include rule.

Reviewer: ```suggestion
Exclude containers from the Agent Autodiscovery perimeter with an exclude rule based on their `name`, `image`, or `kube_namespace` to collect **NO DATA** from these containers. If a container matches an exclude rule, it won't be included unless it first matches an include rule.
```

ac_exclude: [name:<NAME>]
```

> On kubernetes, to remove all containers of pods inside namespace `<NAMESPACE>` from Autodiscovery, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:

Reviewer: ```suggestion
On Kubernetes, to remove all containers of pods inside namespace `<NAMESPACE>` from Autodiscovery, add the following configuration block in the [Agent `datadog.yaml` configuration file][1]:
```
